lib/main.js

```diff
@@ -313,10 +313,11 @@ export default {
 
         let messages;
         if (semver.gte(version, '2.0.0')) {
-          if (!data.files[filePath]) {
+          let fileRealPath = fs.realpathSync(filePath);
```

Comment:
Several things wrong here:

- You haven't brought in `fs`
- `fileRealPath` isn't changed, used `const`
- Don't use the `Sync` method, create a small `async` wrapper and use the correct version.

Reply:
An example of the wrapper function would be something like [this](https://github.com/AtomLinter/linter-jshint/blob/0bf7818f9098d451f674ec273611083b39798032/lib/helpers.js#L63-L71).

Reply:
Thanks @Arcanemagus, I'll do that ASAP. I'm a beginner at this, so could you tell me how to bring `fs`? As I understand it should be treated as semver, minimatch, helpers and path, right? But what should I require?

I'll have to study how to change code and continue with this in the same PR, but I guess it isn't difficult :)

Reply:
> how to bring fs? As I understand it should be treated as semver, minimatch, helpers and path, right?

Exactly, create a variable to act as a placeholder, and then in the `loadDeps` bring it in with `fs = requires('fs');`.